### DISCUSSION TITLE
Double-key CertVerifier

### DISCUSSION THREAD

**Discussion by uazo** [bromite/bromite#2195] [Builds]
can you help me understand the doubts that the chromium team has regarding the inclusion of a Network Isolation also for the cert verifiers?

https://bugs.chromium.org/p/chromium/issues/detail?id=1016890

I find it hard to understand.
Thanks!

**Comment by csagan5** [comment]
I think the Tor browser approach is the best one: https://bugs.chromium.org/p/chromium/issues/detail?id=1016890#c33
That behavior should be enabled by default though. I have read the discussion about shared state (the cache) and what would happen without such cache, there are pros/cons on each from privacy perspective.

Maybe @wchen342 can have a different read.

**Reply by uazo** [reply]
maybe ask the reporter directly?
@arthuredelstein can you help us?

**Reply by wchen342** [reply]
My understanding is the cons are minimal from a privacy perspective (per-site requests can be fingerprinted on network level), the main reason it was not put forward was more non-technical (verification is not part of web platform, no clear spec, etc.).

The behavior is only partially implemented in chromium from the revs though, not sure how difficult it will be to enable it here.

**Reply by uazo** [reply]
it is the same thing that I understood.

> The behavior is only partially implemented in chromium from the revs though

I haven't checked the code thoroughly, but I will. may I ask what do you think is missing?

**Comment by uazo** [comment]
they have already developed it with https://source.chromium.org/chromium/chromium/src/+/88c43a47eb42e904e22d23ba619f1d8a9f14845c
